CMakeLists.txt

```diff
@@ -1,3 +1,56 @@
-cmake_minimum_required(VERSION 3.11.0)
+cmake_minimum_required(VERSION 3.18.0)
 
-# TODO: Write proper build system :)
+project(Pizzas)
+
+
+include(FetchContent)
+FetchContent_Declare(
+    googletest
+    GIT_REPOSITORY https://github.com/google/googletest.git
+    GIT_TAG        main
+)
+FetchContent_MakeAvailable(googletest)
+
+
+find_library(PTHREAD_LIB pthread)
+
+set(SRC_LIST
+    src/Funghi.cpp
+    src/Margherita.cpp
+    src/Pizza.cpp
+    src/Pizzeria.cpp)
```

Comment:
It looks that we have a problem with parser. I have updated the rules for it and now it passes. Your solution is correct.